app/controllers/subscribers_govuk_account_controller.rb

```diff
@@ -0,0 +1,41 @@
+class SubscribersGovukAccountController < ApplicationController
+  before_action :validate_params
+
+  def authenticate
+    account_response = GdsApi.account_api.get_attributes(
+      attributes: %i[email email_verified],
+      govuk_account_session: expected_params[:govuk_account_session],
+    )
+
+    email_verified = account_response.dig("values", "email_verified")
+    email = account_response.dig("values", "email")
+
+    api_response = { govuk_account_session: account_response["govuk_account_session"] }.compact
+
+    render status: :forbidden, json: api_response and return unless email_verified
+    render status: :not_found, json: api_response and return unless email
```

Comment:
Interesting, I like this being here as a defensive measure but is there a scenario where we could actually see this triggered so far?

- we require an email to sign up (currently?)
- so... unless someone has tampered with a payload (in which case `GdsApi::HTTPUnauthorized` anyway?)?

Reply:
I think at the moment this could only trigger if there's a bug in the account manager and so it hasn't sent the attribute to the attribute service or account-api.

Reply:
great to have it there anyway incase we screw something up later.